app/markets/ActivityTable.tsx

```diff
@@ -208,94 +203,94 @@ const TradeRow = ({ activity, isAIAgent }: TradeRowProps) => {
               : '-'}
           </p>
-          <TokenLogo address={activity.collateralToken} className="h-[14px] w-[14px]" />
+          <TokenLogo
+            address={activity.collateralToken}
+            className="h-[13px] w-[13px] opacity-60 hover:scale-110 hover:opacity-100"
+          />
         </div>
-      </TableCell>
-      <TableCell
-        className="text-nowrap pr-4 text-right text-xs text-text-low-em"
+      </div>
+      <div
+        className="text-nowrap text-sm text-text-low-em"
         title={formatDateTimeWithYear(activity.creationTimestamp)}
       >
-        {formatDateTime(activity.creationTimestamp)}
-      </TableCell>
-    </TableRow>
+        {timeAgo(activity.creationTimestamp)}
+      </div>
+    </RowWrapper>
   );
 };
 
+interface AILinkProps {
+  address: string;
+  isAIAgent: boolean;
+}
+
+const AILink = ({ address, isAIAgent }: AILinkProps) => (
+  <div className="flex w-fit flex-shrink-0 items-center space-x-1 text-sm md:text-base">
+    <a
+      href={getGnosisAddressExplorerLink(address)}
+      className={cx(
+        'hover:underline',
+        isAIAgent ? 'text-text-primary-main' : 'text-text-high-em'
+      )}
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      {shortenAddress(address)}
+    </a>
+    {isAIAgent && <Image src="/ai.svg" alt="ai" width={16} height={16} />}
+  </div>
+);
+
 interface LiquidityEventRowProps {
   transaction: FpmmTransaction;
   isAIAgent: boolean;
 }
+
+const RowWrapper = ({ children }: PropsWithChildren) => (
+  <div className="flex w-full min-w-max items-center justify-between space-x-0 p-4">
+    {children}
+  </div>
+);
+
 const LiquidityEventRow = ({ transaction, isAIAgent }: LiquidityEventRowProps) => {
   const creatorAddress = transaction.user.id;
 
   return (
-    <TableRow key={transaction.transactionHash}>
-      <TableCell className="flex items-center space-x-2 pl-4 text-text-high-em">
-        <a
-          href={getGnosisAddressExplorerLink(creatorAddress)}
-          className="hover:underline"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          {shortenAddress(creatorAddress)}
-        </a>
-        {isAIAgent && <Image src="/ai.svg" alt="ai" width={18} height={18} />}
-      </TableCell>
-
-      <TableCell className="truncate text-text-high-em">
-        <div className="flex items-center space-x-2">
-          <p>{formatEtherWithFixedDecimals(transaction.sharesOrPoolTokenAmount)}</p>
-          <Tag size="xs" colorScheme="info" className="w-fit uppercase">
-            {transaction.transactionType}
-          </Tag>
-        </div>
-      </TableCell>
-
-      <TableCell className="truncate text-text-high-em">
-        <div className="flex items-center space-x-1">
+    <RowWrapper>
+      <div className="flex items-center space-x-3">
+        <AILink address={creatorAddress} isAIAgent={isAIAgent} />
+        <div className="flex items-center space-x-1 truncate text-text-high-em">
```

Comment:
Same as here https://github.com/SwaprHQ/presagio/pull/69/files#r1700298511

Let's use " " whitespace instead of `space-x-1`